src/components/landing-page/landing-page.controller.ts

```diff
@@ -2,10 +2,39 @@ import * as angular from 'angular';
 
 
 export class LandingPageController implements angular.IController {
+  static $inject = ['$scope', '$timeout'];
 
   public ctrl: any = this;
+  private $scope: any;
+  private $timeout: any;
+
+  constructor ($scope: any, $timeout: any) {
+    this.$scope = $scope;
+    this.$timeout = $timeout;
+  }
 
   public $onInit() {
     this.ctrl.searchText = '';
   }
+
+  public $postLink() {
+    this.$scope.$watch(() => {
+      return this.ctrl.alerts;
+    }, () => {
+      _.forEach(this.ctrl.alerts, (alert: any) => {
```

Comment:
OK, that makes sense. This was the piece I was most interested in getting a review on. I'm not sure we want every place that add alerts to have to specify how to handle each alert though. What if we made the service configurable to hide alerts by type after the delay? We could also allow individual alerts to specify, but configuring the service would at least be application wide.